src/extensions/userroles.py

```diff
@@ -1,17 +1,13 @@
 import arc
 import hikari
 
+from src.config import ROLE_IDS as role_choices
+role_choices = [hikari.CommandChoice(name=a, value=b) for a,b in role_choices] # reformat role_choices into CommandChoice-es; thus ROLE_IDS can just store easy-to-update tuples
```

Comment:
I don't think it's necessary to import `ROLE_IDS` as something else.
Using full variable names instead of `a` and `b` might also be nice, but that's a pretty optional change imo.